### ISSUE TITLE
how to set my computer on the whitelist for my own node?

### ISSUE THREAD

**Issue by hublex** [getumbrel/umbrel#1728]
![whitebind](https://github.com/getumbrel/umbrel/assets/75634476/3c1f7156-99c3-4ada-a756-e27b8c1bbe60)
how to set my computer on the whitelist? My own node is on a raspberry pi in the same network as my computer.

**Comment by hublex** [comment]
by the way I am using the app bitcoin note. here I couldn't sind any place where I could change the settings reagding whitelist.

**Comment by hublex** [comment]
could it be possible, that it is not working because I have entered the onion address of my bitcoin node?

**Comment by kristapsk** [comment]
> I also had troubles connecting a wallet in the local network, so I've added ultimate rules allowing RPC connections from any IPs

For this purpose you should not use `rpcallowip=0.0.0.0/0`, use `rpcallowip=192.168.0.0/16` or `rpcallowip=10.0.0.0/8`, depending on your LAN settings.

Although likely you are under NAT, so external peers cannot connect directly anyway, unless you add NAT rule (port forwarding) for `tcp/8332` RPC port from your external IP to LAN IP of the node in the router. But you might in future connect the same node to different network and forget about this, so better is to use more restrictive config.

**Comment by kristapsk** [comment]
Of course, if you want to access node RPC from a single computer in LAN with static IP, then just add `rpcallowip` for that single IP, not whole network.

**Comment by Radiokot** [comment]
Many thanks, @kristapsk 🤝

**Comment by hublex** [comment]
Hi Guys, Art for the nood questions?
Where I can change the settings, please?
Cheers

**Comment by Radiokot** [comment]
@hublex [Connect to your node through SSH](https://thebitcoinmachines.com/ssh-into-umbrel-machine/) and do the described changes.
